src/pages/gateway/command-reference.md

```diff
@@ -257,12 +257,17 @@ aio api-mesh:describe
 ### Response
 
 ```terminal
-Successfully retrieved mesh details
+Selected organization: my-org
+Selected project: test-project
+Select workspace: Stage
+Successfully retrieved mesh details 
 
-Org ID: 123456
+Org ID: 123456789
 Project ID: 1234567890123456789
 Workspace ID: 2345678901234567890
 Mesh ID: 12a3b4c5-6d78-4012-3456-7e890fa1bcde
+API Key: f0a3b4c56d78401234567e890fa1bcde
+Mesh Endpoint: https://graph.adobe.io/api/12a3b4c5-6d78-4012-3456-7e890fa1bcde/graphql?api_key=12a3b4c5-6d78-4012-3456-7e890fa1bcde
```

Comment:
I am hoping this isnt a live mesh id and api key?

Reply:
No, it is just `abc...` and `123...` mixed together